locallib.php

```diff
@@ -163,12 +163,14 @@ function accredible_issue_default_certificate($userid, $certificateid, $name, $e
         $response = $restapi->create_evidence_item_grade($grade, $quizname, $credentialid, $hidden);
     }
 
+    $evidenceitem = new evidenceitem();
```

Comment:
This will throw a not found exception as the class `evidenceitems` are not included in this file.
Add - `use mod_accredible\local\evidenceitems;`